BREAKING CHANGE(web-react): Rename RadioField and CheckboxField to Radio and Checkbox

crishpeen opened the pull request:
Description
Renamed components CheckboxField and RadioField to Checkbox and Radio

Issue reference
#DS-521, #DS-522

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md).
- [x] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

literat commented on packages/web-react/src/components/Checkbox/stories/CheckboxProps.tsx:
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { RadioField } from '../RadioField';
```
Interesting 🤔

literat replied:
Oh wait! Are we really renaming `RadioFieldProps.tsx` to `CheckboxProps.tsx` or this is some kind of Git magic?

literat commented on packages/web-react/src/components/Radio/stories/RadioDisabled.tsx:
`RadioFieldItem.tsx` -> `RadioDisabled.tsx` 🤔

literat commented on packages/web-react/src/components/Radio/stories/RadioItem.tsx:
Same as above.

literat reviewed (changes requested):
Please check if the files are correctly renamed or if there is some Git black magic.

crishpeen commented:
@literat check the code changes by commit. Then you can see it is just the wrong interpretation of GitHub because I do not rename radio<-->checkbox. Because I changed the name of the folders it somewhere decided I removed a file and then created a new one because the path changed. 

Imho everything is as it should be. :)
